pkgs/development/python-modules/markdown2/default.nix

```
@@ -1,15 +1,23 @@
-{ lib, buildPythonPackage, fetchPypi }:
+{ lib, buildPythonPackage, fetchFromGitHub, pygments }:
 
 buildPythonPackage rec {
   pname = "markdown2";
-  version = "2.3.6";
+  version = "2.4.0";
 
-  src = fetchPypi {
-    inherit pname version;
-    extension = "zip";
-    sha256 = "08a124043aa0ad36ba2136239547d5011a2b770278abb11a5609611e0040ea05";
+  # PyPI does not contain tests, so using GitHub instead.
+  src = fetchFromGitHub {
+    owner = "trentm";
+    repo = "python-markdown2";
+    rev = "${version}";
+    sha256 = "sha256:03qmf087phpj0h9hx111k4r5pkm48dhb61mqhp1v75gd09k0z79z";
   };
 
+  checkInputs = [ pygments ];
+
+  checkPhase = ''
+    python ./test/test.py
```

to support alternative python interpreters such as pypy
```suggestion
    ${python.interpreter} ./test/test.py
```

---

Thanks! I've force pushed a new commit to include your suggestions